webhook/resources/volume/validator.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 
+	apiequality "k8s.io/apimachinery/pkg/api/equality"
 	"k8s.io/apimachinery/pkg/api/resource"
 	"k8s.io/apimachinery/pkg/runtime"
```

Comment:
```suggestion
"k8s.io/apimachinery/pkg/api/resource"
"k8s.io/apimachinery/pkg/runtime"

apiequality "k8s.io/apimachinery/pkg/api/equality"
```

Reply:
We have a convention like https://github.com/longhorn/longhorn/wiki/coding-convention#organize-package-imports

Reply:
Ok. Fixed.

Btw, [the convention](https://github.com/longhorn/longhorn/wiki/coding-convention#organize-package-imports) has a typo at 
```go
		// third-party packages without alias names related to k8s
		corev1 "k8s.io/api/core/v1"

		// third-party packages with alias names related to k8s
		"k8s.io/kubernetes/pkg/controller"

```